examples/server_certificate_manager.py

```diff
@@ -0,0 +1,53 @@
+from opcua import ua, Server
+from opcua.crypto import uacrypto
+
+import time
+
+import sys
+sys.path.insert(0, "..")
+
+
+def simple_certificate_manager(isession, certificate, signature):
```

Comment:
I have not looked in detail at this. But is it enough with a function? Should'nt we use a class with a few methods? just asking

Reply:
if it is just a function, then using the name manager is a bit wrong. A manager is an object.  a function would be called something like set_certificate_validation_function or whatever